spec/windows.dd

```diff
@@ -0,0 +1,46 @@
+Ddoc
+
+$(SPEC_S Windows Programming,
+
+$(HEADERNAV_TOC)
+
+$(P This covers Windows programming with 32 bit OMF, 32 bit Mscoff,
+64 bit Mscoff, console programs, GUI programs, and DLLs.
+)
+
+$(H2 $(LNAME2 mscoff, Windows 32 and 64 bit MSCOFF Programs))
+
+    $(P 32 bit and 64 bit MSCOFF programs use the Microsoft Visual C/C++ compiler as the Associated
+    C Compiler, generate object files in the MSCOFF format, and use the Microsoft linker
+    to link them.
+    )
+
+$(H3 $(LNAME2 mscoff-console, Console Programs))
+
+$(H3 $(LNAME2 mscoff-windows, Windows GUI Programs))
+
+$(H3 $(LNAME2 mscoff-dlls, DLLs))
+
+$(H2 $(LNAME2 omf, Windows 32 bit OMF Programs))
+
+    $(P 32 bit OMF programs use the
+    $(LINK2 https://www.digitalmars.com/download/freecompiler.html, Digital Mars C/C++ compiler)
+    as the Associated C Compiler, generate object files in the OMF format, and use the
+    $(LINK2 https://www.digitalmars.com/ctg/optlink.html, Optlink linker)
+    to link them.
+    )
+
+$(H3 $(LNAME2 omf-console, Console Programs))
+
+$(H3 $(LNAME2 omf-windows, Windows GUI Programs))
+
+$(H3 $(LNAME2 omf-dlls, DLLs))
```

Comment:
That'll need to cover linker scripts which (for some reason) are required to generat a dll (at least last time I tried -shared wasn't enough).

Reply:
I know. This page is currently just an outline.

Reply:
All good, just noting it from experience :)